lib/ServiceEmitter.js

```diff
@@ -31,6 +31,15 @@ class ServiceEmitter extends EventEmitter {
     this._delay = opts.delay || 1000;
 
     /**
+     * Determines if is requesting data immediately
```

Comment:
Change "Determines if is requesting data immediately" for "Determines if {@link this._intervalCallback()} should be called immediately".